libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -327,7 +327,7 @@ void AC_Autorotation::update_forward_speed_controller(float pilot_accel_norm)
 
     // Calculate roll and pitch targets from angles, negative accel for negative pitch (pitch forward)
     Vector2f angle_target_deg = { accel_mss_to_angle_deg(-bf_accel_target_mss.x), // Pitch
-                              accel_mss_to_angle_deg(bf_accel_target_mss.y)}; // Roll
+                                  accel_mss_to_angle_deg(bf_accel_target_mss.y)}; // Roll
```

Comment:
might as well drop this change so AC_Autorotation is untouched, there are no other changes for this lib.

Reply:
Done